lte/gateway/fabfile.py

```diff
@@ -188,7 +188,7 @@ def s1ap_setup_cloud():
 
 
 def integ_test(gateway_host=None, test_host=None, trf_host=None,
-               destroy_vm="True"):
+               destroy_vm='True', provision_magma='True'):
```

Comment:
why not `provision_vm`? seems better naming for the param (similar to `destroy_vm`)